components/icon.vue

```diff
@@ -5,32 +5,60 @@
 </template>
 
 <script>
+
+import _ from 'lodash'
+
+// Extend this list when you need a new icon.
```

Comment:
I understand why this changed, but make it into a TODO to make it prettier again

Reply:
I just removed prettier....
I replaced it by file list of icons.